network-api/networkapi/buyersguide/templates/catalog.html

```diff
@@ -0,0 +1,110 @@
+            {% endblocktrans %}
+          </p>
+        </div>
+      </div>
+    </div>
+</div>
+
+<div class="project-list-section">
+  <div class="creepiness-slider bg-white text-center">
+    <span class="current-creepiness">
+      <span class="speech-bubble d-none">
+        <span class="text"></span>
+        <span class="hook"></span>
+      </span>
+    </span>
+  </div>
+
+  <div class="product-box-list-wrapper">
+    <div class="product-box-list d-flex justify-content-center align-items-stretch flex-wrap">
+
+      {% for product in products %}
+        {% if product.is_current %}
+
+        <figure class="product-box d-flex flex-column justify-content-between{% if product.draft %} draft-product{% endif %}{% if product.adult_content %} adult-content{% endif %}" data-creepiness="{{ product.votes.creepiness.average }}">
+          <div class="top-left-badge-container">
+              {% include "fragments/seal_of_approval.html" with product=product %}
+              {% if product.votes.confidence %}
+                  {% if product.votes.confidence.1 > product.votes.confidence.0 %}
+                      <div class="d-none recommendation positive"></div>
+                  {% else %}
+                      <div class="d-none recommendation negative"></div>
+                  {% endif %}
+              {% endif %}
+          </div>
+
+          {% include "fragments/adult_content_badge.html" with product=product %}
+
+          <a class="product-image text-center mt-4 h-100 d-flex flex-column justify-content-between" href="{% url 'product-view' product.slug %}">
+              <img
+                class="product-thumbnail"
+                {% if USE_CLOUDINARY %}
+                  src="{% cloudinary_url product.cloudinary_image quality=50 fetch_format="auto" crop="fit" width=600 %}"
+                {% else %}
+                  src="{{mediaUrl}}{{"AWS_LOCATION"|env}}/{{product.image }}"
+                {% endif %}
+                alt="{% blocktrans with product=product.name %}link to {{product}}{% endblocktrans %}"
+              >
+          </a>
+
+          <figcaption class="d-block mt-md-2 text-left">
+            <a class="product-links" href="{% url 'product-view' product.slug %}">
+              <div class="product-company body-small">{{product.company}}</div>
+              <div class="product-name body">{{product.name}}</div>
+            </a>
+          </figcaption>
+        </figure>
+        {% endif %}
+      {% endfor %}
+    </div>
+  </div>
+</div>
+<div class="recommend-product">
+ <div class="container text-center my-5">
+  <h2 class="h3-heading">{% trans "Don’t see the product you’re looking for?" %}</h2>
+  <p class="body">{% trans "Let us know what product you would like reviewed in the guide." %}</p>
+  <a href="https://forms.gle/UefDgjEU85Xwcf8C8" target="_blank" class="btn btn-secondary btn-recommend">{% trans "Submit a product" %}</a>
+ </div>
```

Comment:
Let's put this in a `{% block recommend_product %} ... {% endblock %}` so that we can make that an empty block for category pages, since only the homepage has that block?

Reply:
Good point! I will check with Sabrina.

Reply:
@Pomax I checked with Sabrina and Jen - we want this block to show on both homepage and category pages except the search is on the "no results" state.

Reply:
awesome, in that case this is R+ =D